src/nft/types/navbar/navbar.ts

```diff
@@ -1,28 +1,48 @@
+import { NATIVE_CHAIN_ID } from 'constants/tokens'
+import { SafetyLevel } from 'graphql/data/__generated__/types-and-hooks'
+import { SearchToken } from 'graphql/data/SearchTokens'
+import { CHAIN_NAME_TO_CHAIN_ID } from 'graphql/data/util'
+
 export interface LooksRareRewardsData {
   address: string
   cumulativeLooksAmount: string
   cumulativeLooksProof: string[]
 }
-interface BridgeInfoEntry {
-  tokenAddress?: string
-}
-
-interface FungibleTokenExtensions {
-  bridgeInfo?: { [chain: number]: BridgeInfoEntry }
-}
 
 export interface FungibleToken {
```

Comment:
This type is used outside of nfts it should probably be moved outside of the nft folder. Or possibly deprecate this type and use the generated type with the additional of the isVerified logic

Reply:
Yes would love to get rid of this type, but I need to migrate trending tokens and recently searched tokens first too (those PR's are next).

Reply:
FungibleToken is removed in #5896